src/tests/dbus-tests/test_60_partitioning.py

```diff
@@ -474,6 +474,30 @@ def test_dos_flags(self):
         _ret, sys_flags = self.run_command('blkid /dev/%s -p -o value -s PART_ENTRY_FLAGS' % part_name)
         self.assertEqual(sys_flags, '0x80')
 
+    def test_gpt_flags(self):
+        disk = self.get_object('/block_devices/' + os.path.basename(self.vdevs[0]))
+        self.assertIsNotNone(disk)
+
+        self._create_format(disk, 'gpt')
+        self.addCleanup(self._remove_format, disk)
+
+        part = self._create_partition(disk, fmt=False)
+        self.addCleanup(self._remove_partition, part)
+
+        # set legacy BIOS bootable flag (100(2), 4(10), 0x4(16))
+        part.SetFlags(dbus.UInt64(100), self.no_options,
```

Comment:
This should be ``dbus.UInt64(4)``, ``100`` works too, but not because it is treated as a binary form, instead:
```
In [1]: bin(100)
Out[1]: '0b1100100'
```